app/views/hyrax/my/_facet_limit.html.erb

```diff
@@ -1,4 +1,4 @@
-  <ul class="dropdown-menu facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
+  <ul class="<%= 'dropdown-menu ' if action_name == 'index' %>facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
```

Comment:
I'd like to see a comment added here explaining why we need this check. Otherwise someone might come along and remove this in the future because they don't understand it's purpose.